internal/runners/pvc_autoresizer.go

```diff
@@ -98,17 +94,15 @@ func (w *pvcAutoresizer) getStorageClassList(ctx context.Context) (*storagev1.St
 	return &scs, nil
 }
 
-func (w *pvcAutoresizer) reconcile(ctx context.Context) error {
+func (w *pvcAutoresizer) reconcile(ctx context.Context) {
 	scs, err := w.getStorageClassList(ctx)
 	if err != nil {
 		w.log.Error(err, "getStorageClassList failed")
-		return nil
```

Comment:
When we fail to get the storage class, we cannot go on. `return` is necessary.

Reply:
Thanks for your feedback. I fixed my code. Could you review this again?